sk_SK.lang

```diff
@@ -0,0 +1,347 @@
+replaymod.title=Replay Mód
+
+#All of the chat messages
+replaymod.chat.recordingstarted=Nahrávanie začalo
+replaymod.chat.recordingfailed=Pri nahrávaní nastala chyba
+
+replaymod.chat.savingthumb=Ukladám náhľad...
+replaymod.chat.savedthumb=Náhľad bol uložený
+replaymod.chat.failedthumb=Náhľad nemohol byť uložený
+
+replaymod.chat.addedmarker=Event Marker has been added
+
+#Chat messages displayed in Replay Viewer
+replaymod.chat.morekeyframes=Aspoň 2 kľúčové pozície a snímky sú povinné
+replaymod.chat.pathstarted=Cesta kamery začala
+replaymod.chat.pathfinished=Cesta kamery skončila
+replaymod.chat.pathinterrupted=Cesta kamery bola ukončená
+
+#Common GUI-related strings
+replaymod.gui.cancel=Zrušiť
+replaymod.gui.back=Späť
+replaymod.gui.load=Načítať
+replaymod.gui.removelike=Odstrániť lajk
+replaymod.gui.removedislike=Odstrániť dislajk
+replaymod.gui.save=Uložiť
+replaymod.gui.rename=Premenovať
+replaymod.gui.remove=Odstrániť
+replaymod.gui.delete=Vymazať
+replaymod.gui.paused=Zastavené
+replaymod.gui.speed=Rýchlosť
+replaymod.gui.pleasewait=Prosím počkajte
+replaymod.gui.render=Vykresliť
+replaymod.gui.iphidden=Adresa servera skrytá
+replaymod.gui.overwrite=Prepísať
+replaymod.gui.saveas=Uložiť ako...
+replaymod.gui.done=Hotovo
+replaymod.gui.close=Zavrieť
+replaymod.gui.notagain=Neukazovať znovu #vanilla consistency
```

Comment:
in vanilla this line is in the game and i want to use that, for now there is prototype line

Reply:
cant find the string... will ask for it